src/components/MainFeed/Comments.jsx

```diff
@@ -0,0 +1,8 @@
+export function Comments() {
+  return (
+    <div className="commentsContainer">
+      <p>This is a very very very vert long comment in case you wanted to know
+      </p>
+    </div>
+  );
+}
```

Comment:
I think that the Comments component should still accept a prop, so that it can be reusable!

maybe something like this:

```suggestion
export function Comments(props) {
  return (
    <div className="commentsContainer">
      <p>{props.postComments}</p>
    </div>
  );
}
```